roles/haproxy/tasks/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
 - name: Install haproxy
   action: "{{ ansible_pkg_mgr }} name=haproxy state=present"
-  when: not openshift.common.is_containerized | bool
```

Comment:
Okay I'll remove it, it'll work with the `openshift_facts` addition in `playbooks/common/openshift-master/config.yml`